fetch.bs

```diff
@@ -6224,17 +6224,19 @@ constructor must run these steps:
   <p>If <var>inputBody</var> is <var>body</var> and <var>inputBody</var> is non-null, then:
 
   <ol>
-   <li>
-    <p>Let <var>rs</var> bs a {{ReadableStream}} object from which one can read the exactly
-    same data as one could read from <var>inputBody</var>'s <a for=body>stream</a>.
+   <li><p>Let <var>ws</var> and <var>rs</var> be the
+   <a href="https://streams.spec.whatwg.org/#writable-side">writable side</a> and
+   <a href="https://streams.spec.whatwg.org/#readable-side">readable side</a> of an
+   <a href="https://streams.spec.whatwg.org/#identity-transform-stream">identity transform
+   stream</a> respectively.</li>
```

Comment:
I think it would be better to have a comma before "respectively", but not 100% confident of this.

Reply:
Fixed.